accepted/package-importer.d.ts.md

```diff
@@ -290,19 +296,22 @@ any URL with the `pkg:` scheme. This step will be inserted immediately before
 the existing legacy importer logic, and if the package importer returns `null`,
 the legacy importer logic will be invoked.
 
-Currently, the only available package importer is `node`, which follows Node
-resolution logic to locate Sass files.
+Currently, the only available package importer is `NodePackageImporter`, which
+follows Node resolution logic to locate Sass files.
 
-An optional `entryPointPath` path can be passed to provide a starting
-`parentURL` for the Node package resolution algorithm. If not set, the default
-value is `require.main.filename`.
+An optional `entryPointDirectory` path can be passed to the
+`NodePackageImporter` to provide a starting `parentURL` for the Node package
+resolution algorithm. If not set, the default value is the parent directory of
+`require.main.filename`.
 
 Defaults to undefined.
 
 ```ts
+import {NodePackageImporter as BaseNodePackageImporter} from '../spec/js-api/importer';
+
 declare module '../spec/js-api/legacy/options' {
   export interface LegacySharedOptions<sync extends 'sync' | 'async'> {
-    pkgImporter?: NodePackageImporter;
+    pkgImporter?: BaseNodePackageImporter;
```

Comment:
@nex3 This workaround was required by the new symbol key, so that the two definitions of `pkgImporter` are now actually using the same definition of `NodePackageImporter`.